source/clear-linux/get-started/get-started.rst

```diff
@@ -33,4 +40,10 @@ Install |CL|
    virtual-machine-install/virtual-machine-install
    live-image
    compatibility-check
-   bootable-usb/bootable-usb
\ No newline at end of file
+   bootable-usb/bootable-usb
+   bare-metal-install-beta-intro/bare-metal-install-beta-intro
+
+   bare-metal-install-beta/bare-metal-install-beta
+   bootable-usb/bootable-usb-beta-all
+
+..After new TUI installer is incorporated in main dir, remove final two entries above. 
```

Comment:
Comment on line 49 - needs to have single space between ".." and "After" in order to be read as a comment by Sphinx. As is, it will render to the final html.

Reply:
Thank you.